### PR TITLE
Manage teams apps on Office mobile

### PR DESCRIPTION
**Pull request by harshatms**
Allow, block teams apps on office mobile.

### CODE REVIEW

File: memdocs/intune/apps/manage-microsoft-office.md [hunk 1]
```diff
@@ -145,6 +146,22 @@ If you need to enable or disable the Office Store portion of the platform for iO
...
 
+### Manage Teams apps running on Office mobile on Android (in Preview)
+
+Teams personal tab apps are already in Preview for Office.com and Outlook desktop in Target Release, and IT admins can manage access to Teams apps by creating [custom permission policies](https://docs.microsoft.com/en-us/MicrosoftTeams/teams-app-permission-policies#create-a-custom-app-permission-policy) and [assigning these policies to users](https://docs.microsoft.com/en-us/MicrosoftTeams/policy-assignment-overview) using Teams Admin Center. You can now also run Teams personal tab apps in Office mobile on Android, in Preview. Teams personal tab apps built using [Microsoft Teams JavaScript client SDK v2](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit) (version 2.0.0) and [Teams App manifest](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema) (version 1.13) will start showing in Office mobile Android app under the “Apps” menu.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/manage-microsoft-office.md:151]:
```suggestion
Teams personal tab apps are already in preview for Office.com and Outlook desktop in Target Release, and IT admins can manage access to Teams apps by creating [custom permission policies](/MicrosoftTeams/teams-app-permission-policies#create-a-custom-app-permission-policy) and [assigning these policies to users](/MicrosoftTeams/policy-assignment-overview) using Teams admin center. You can now also run Teams personal tab apps in Office mobile on Android, in preview. Teams personal tab apps built using [Microsoft Teams JavaScript client SDK v2](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit) (version 2.0.0) and [Teams App manifest](/microsoftteams/platform/resources/schema/manifest-schema) (version 1.13) will appear in Office mobile Android app under the “Apps” menu.
```

File: memdocs/intune/apps/manage-microsoft-office.md [hunk 2]
```diff
@@ -145,6 +146,22 @@ If you need to enable or disable the Office Store portion of the platform for iO
...
+
+There may be additional management requirements specific to Office mobile on Android. You may want to:
+- Only allow specific users in your org to try enhanced Teams apps on Office mobile, or
+- Block all users in your org from using enhanced Teams apps on Office mobile.
+
+To manage the above, you can use the following key:
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/manage-microsoft-office.md:157]:
```suggestion
- Only allow specific users in your organization to try enhanced Teams apps on Office mobile, or
- Block all users in your organization from using enhanced Teams apps on Office mobile.

To manage these,  you can use the following key:
```

File: memdocs/intune/apps/manage-microsoft-office.md [hunk 3]
```diff
@@ -145,6 +146,22 @@ If you need to enable or disable the Office Store portion of the platform for iO
...
+
+|    Key    |    Value    |
+|-------------------------------------------------------------------|-------------|
+|    com.microsoft.office.officemobile.TeamsApps.IsAllowed    |    **true** (default) enables Teams apps on Office mobile<br>**false** disables Teams apps on Office mobile    |
+
+Above can also be used both by managed devices and managed apps.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/manage-microsoft-office.md:163]:
```suggestion
This key can also be used both by managed devices and managed apps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @harshatms, kindly commit the suggested changes. Many thanks.

File: memdocs/intune/apps/manage-microsoft-office.md [hunk 4]
```diff
@@ -145,6 +146,22 @@ If you need to enable or disable the Office Store portion of the platform for iO
 
 For more information about adding configuration keys, see [Add app configuration policies for managed iOS/iPadOS devices](../apps/app-configuration-policies-use-ios.md).
 
+### Manage Teams apps running on Office mobile on Android (in Preview)
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/manage-microsoft-office.md:149]:
```suggestion
### Manage Teams apps running on Office mobile on Android (in preview)
```

File: memdocs/intune/apps/manage-microsoft-office.md (hunk 3 above)
Comment (harshatms) [reply memdocs/intune/apps/manage-microsoft-office.md]: Removed "also".

**Comment by harshatms** [comment]
@JohanFreelancer9 I have resolved comments.

**Review by JohanFreelancer9 — APPROVED** [review]

File: memdocs/intune/apps/manage-microsoft-office.md [hunk 5]
```diff
@@ -120,6 +120,7 @@ Office supports the following settings for configuration:
 
 - Manage the creation of Sticky Notes
 - Set add-ins preference
+- Manage Teams apps running on Office app for Android (in Preview)
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/manage-microsoft-office.md:123]:
```suggestion
- Manage Teams apps running on Office app for Android (in preview)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Thank you for committing the suggested changes. Yes, the relative links will definitely work once these changes have been merged. And my apologies, could you please also commit the one that I've missed? Thanks again.

**Comment by harshatms** [comment]
> Thank you for committing the suggested changes. Yes, the relative links will definitely work once these changes have been merged. And my apologies, could you please also commit the one that I've missed? Thanks again.

Thanks for clarification. I have committed suggestions.

**Comment by Erikre** [comment]
#sign-off

**Comment by harshatms** [comment]
@Erikre and @JohanFreelancer9 how can I merge this PR?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@yogkumgit, can you please help to get this PR merged? Many thanks.

**Comment by harshatms** [comment]
> @yogkumgit, can you please help to get this PR merged? Many thanks.

**Comment by harshatms** [comment]
@yogkumgit kindly help in merging this.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**